apps/web/src/components/activity/ExecutionDetail.tsx

```diff
@@ -0,0 +1,112 @@
+import { Container, Grid, useMantineColorScheme } from '@mantine/core';
+import { ExecutionDetailsStatusEnum, StepTypeEnum } from '@novu/shared';
+import { format, parseISO } from 'date-fns';
+import styled from 'styled-components';
+
+import { ExecutionDetailShowRaw } from './ExecutionDetailShowRaw';
+
+import { Button, colors, Text } from '../../design-system';
+import { CheckCircle, Digest, ErrorIcon, Mail, Timer } from '../../design-system/icons';
+
+const DetailText = styled(Text)<{ theme: string }>`
```

Comment:
These are the default style settings for `<Text>`. I believe DetailText is not needed here 

Also theme could be of type MantineTheme as you use theme.colorScheme, otherwise you could pass colorScheme which is a string of 'light'|'dark'

Reply:
I have been moving back and forth from theme to colorScheme. I finally went for theme because colorScheme was being rejected by React as a non-DOM property and the way we have our wrapper implementation of Text was helping to that error.
Anyway, I will remove the styled component as it is not needed. Just double checked that the theme rendering works as expected. Thanks for the advise in this.